app/controllers/root_controller.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
 class RootController < ApplicationController
-  def index; end
+  def index
+    @articles = Article.eager_load(:user).reorder(created_at: :DESC).all if user_signed_in?
```

Comment:
it is usually lowercase
```suggestion
    @articles = Article.eager_load(:user).reorder(created_at: :desc).all if user_signed_in?
```